blueocean-bitbucket-pipeline/src/test/resources/api/server/mappings/mapping-branches-default-C98AD.json

```diff
@@ -1,5 +1,5 @@
 {
-  "id" : "3cb8ad8e-bf64-3779-b249-1c1ca33c783a",
+  "id" : "c8935a42-83a8-48dd-8a37-cd5fdbdfd91d",
```

Comment:
Here and elsewhere the new version of Wiremock doesn't allow duplicate UUIDs in mapping files. Since several of our mapping files contained the same UUID and failed to load in the new version of Wiremock, I resolved this by choosing a new UUID to ensure each file had a unique UUID.